src/styles/sidebar/components/menu.scss

```diff
@@ -44,9 +44,9 @@
   // overlaps the content's border. The effect is that the menu's border is a
   // rounded rect with a notch at the top.
   position: absolute;
-  top: calc(100% - 2px);  // nb. Adjust this if changing the <svg> size.
+  top: calc(100% - 2px); // nb. Adjust this if changing the <svg> size.
   right: 0;
-  z-index: 1;
+  z-index: 2;
```

Comment:
Note these `z-index` changes. These fix the problem where the menu's content div is below some content in an annotation card, but I am not sure this is kosher?